Makefile

```diff
@@ -76,7 +76,7 @@ debug: all
 release: all
 	./build-qemu.sh $@		
 
-tests: release install x86_tests a64_tests
+tests: release install a64_tests
```

Comment:
Why is this change necessary? Also what is the binary file you are adding in this commit?